business-loyalty-sdk/src/main/java/co/nimblehq/loyalty/sdk/LoyaltySdk.kt

```diff
@@ -71,6 +68,10 @@ class LoyaltySdk private constructor() : NetworkBuilder() {
         }
     }
 
+    /***********
+     * LOYALTY *
+     ***********/
```

Comment:
should be Reward. 
if we talk about a domain like `Loyalty` it doesn't on par with the below `Product` section